src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -714,9 +714,12 @@ private void declareLegacyNamespaceAliases() {
           googModuleStyleName = e.getKey();
         }
         emitNamespaceBegin(namespace);
-        // TODO(lucassloan): not technically correct, since this emits an alias as if it's a class
-        // with a type and value. Change to emit the alias only for type or value as necessary.
-        visitKnownTypeValueAlias(googModuleStyleName, e.getValue());
```

Comment:
I have concerns that previously we could emit here purely based on the name, whereas now we require type information. If there is a case where the type info is not present for some reason we could potentially regress here. 

That said, I don't have a good idea how could this happen, so let's see what happens during presubmit.
No action requred.

Reply:
Yep, needs a presubmit.